catalog/task/add-bundles-to-image-index/0.1/add-bundles-to-image-index.yaml

```diff
@@ -0,0 +1,93 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: add-bundles-to-image-index
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+      Submit a build request to add operator bundles to an index image
+  params:
+    - name: IIB_URL 
+      description: IIB service url
+      type: string
+    - name: IIB_JSON_BUILD_ARGS
+      description: IIB JSON arguments to add a new build
+      type: string
+    - name: IIB_SECRET
+      type: string
+      description: Secret with iib credentials to be used
+  results:
+    - name: json_build_info
+      description: JSON build information for the requested build
+    - name: build_state
+      description: IIB Service build state
+  steps:
+    - name: add-bundles-to-image-index
+      image: quay.io/hacbs-release/release-utils
```

Comment:
I know other tasks don't, but I wonder if we should be adding a tag here. @davidmogar thoughts?

Reply:
We should use tags everywhere. This depends on having a latest one.